codegen/templates/service.tmpl

```diff
@@ -38,9 +38,11 @@ func CreateGateway(
 }
 
 func registerDeps(g *zanzibar.Gateway, deps *module.Dependencies) error {
+	var err error
```

Comment:
instead can you use named response variable?

`func registerDeps(g *zanzibar.Gateway, deps *module.Dependencies) (err error)`

Reply:
Hmmm, I have always wondered why some prefer named return variables. To me, when I see a return with no value, I have to go back to the function signature to figure out what's returned, and then I need to find out where that named return variable is set, seems too much hassle.

Reply:
you dont need to use named variables with empty return. It avoids the variable declaration, thats all.